website/docs/getting-started/tutorials/customer-support-tool/build-the-dashboard.md

```diff
@@ -0,0 +1,190 @@
+---
+sidebar_position: 1
+---
+
+
+# Build the Dashboard
+
+In this section, you'll learn to:
+* Connect to the tickets database.
+* Fetch the ticket details from the database.
+* Build a dashboard using the Chart and Stats Box widget.
+* Display data from queries on the dashboard.
+
+## Connect the database
+
+1. On the *Entity Explorer* to the left of the screen, click on the **Dashboard** page you created in the [Setting up the application](/getting-started/tutorials/customer-support-tool#setting-up-the-application) section.
+
+2. In the **Explorer** tab, click the **+** icon next to **Datasources** to add a new datasource.
+
+3. Select **PostgreSQL** under the **Databases** section. This opens the page where you can configure the fields to connect to a PostgreSQL database. 
+
+4. Click the pencil icon next to the default database name on the top left and rename the database to `supportTickets`.
+
+5. Enter the following details in the connection parameter fields:<br/>
+    - **Host Address**: `mockdb.internal.appsmith.com` <br/>
+    - **Port**: `5432`<br/>
+    - **Database Name**: `mockdb_v2`<br/>
+    - **Username**: `postgres`<br/>
+    - **Password**: `plenty-cape-quality`<br/>
+
+6. Click the **Test** button to test the connection and ensure the database is valid.
```

Comment:
Let's write what user sees when a database is valid. As users wouldn't know how the database could be valid (`ensure the database is valid`).

Reply:
This may change in the UI from time to time as it is a toast message.

Reply:
Can we rewrite in a way so that it's shows that we are teaching them how this is done rather than relying on them to validate. Click the **Test** button to verify that the database connection has been successfully established.",